### ISSUE TITLE
Incompatibility with HDT-SMP

### ISSUE THREAD

**Issue by LostDragonist** [ModOrganizer2/modorganizer#704]
User report from sidfu#6438 on Discord.

## The problem:

https://cdn.discordapp.com/attachments/403979692501106688/574388810041786368/Motoko_-_Aretino_Residence_2019-05-04_19.01.28.jpg

User reports problem due to HDT-SMP on Skyrim LE.  Works with MO 2.1.6.  Does not work with MO 2.2.0.

User gave the following links as instructions on how to install HDT-SMP **(NSFW)**: https://www.loverslab.com/topic/68009-all-in-one-hdtskinnedmeshphysics-setup-20b-fomod/

## Environment:

* Mod Organizer Version that exhibits the issue: 2.2.0
* Last Mod Organizer Verison that did not exhibit the issue (if applicable): 2.1.6
* Desktop OS/version used to run Mod Organizer: ???

**Comment by isshininu** [comment]
Keep in mind that those DLLs are from 2016 and outdated.
Most users use updated DLLs **smp-pe path 0730**, that _unofficially_ provided as a patch for Yundao HDT hairs. https://www.nexusmods.com/skyrim/mods/88416?tab=files (SFW)

**Comment by OrrieL** [comment]
MO2 2.2.0 breaks HDT-SMP for me. The game CTDs on save load (even tried coc qasmoke from the menu, same CTD)
Rolling back to MO2 2.1.6 and the save loads fine.
Skyrim LE, HDT-SMP DLL (hdtSkinnedMeshPhysics.dll 910336 bytes CRC32 8ECBD2A2 which is I believe the latest available and it is the one isshininu points out in previous comment).
Win 8.1 64b

**Comment by Al12rs** [comment]
Can anyone report whether the recent usvfs patch on discord still reproduces this issue? https://discord.gg/6GKR9jZ

**Comment by OrrieL** [comment]
> Can anyone report whether the recent usvfs patch on discord still reproduces this issue? https://discord.gg/6GKR9jZ

I tried the latest usvfspatch2 from discord with MO2 2.2.0 release. Still the same issue.

smp dll in usvfs + MO2.1.6 - OK
smp dll in game data folder + MO2.2.0 - OK
smp dll in usvfs + MO2.2.0 - CTD on save load
smp dll in uscfs + MO2.2.0+usvfspath2 - CTD on save load

**Comment by Al12rs** [comment]
Thanks for testing it, could you submit your debug logs of the issue while using the new usvfs patch?

Instructions fro logs:

To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

Before sending any Logs **make sure the Log Level is set to Debug** in Settings\Diagnostics, then close MO2 and clean the logs folder, after that you can reopen MO2 and reproduce the problem. 

Once you have reproduced the problem, close MO2 before sending or examining the logs as they are finished writing only after the program is closed; after you have finished testing setting the log level back to default is recommended.

You should always send the mo_interface.log file as that one contains the general information of your setup, for the usvfs log, you should upload the latest one if the issue is about programs started through Mo2.

You can drag and drop the logs directly on github to send them (tip: zip them if they are too big).

 https://cdn.discordapp.com/attachments/458597400638914563/458728891620524032/unknown.png

**Comment by OrrieL** [comment]
[MOlogs.zip](https://github.com/ModOrganizer2/modorganizer/files/3335485/MOlogs.zip)
Used minimal loadorder.
Skyrim.esm
Update.esm
Dawnguard.esm
HearthFires.esm
Dragonborn.esm
Unofficial Skyrim Legendary Edition Patch.esp
XPMSE.esp
FNIS.esp
RaceMenu.esp
RaceMenuPlugin.esp
+ENB
+SMP dll
launched the game via MO2, in the main menu from console "coc qasmoke", loading, CTD.

**Comment by Al12rs** [comment]
Does this file exist in virtual data? `C:\Games\Steam\steamapps\common\skyrim\DATA\SKSE\Plugins\hdtSkinnedMeshConfigs\defaultBBPs.xml`

**Comment by OrrieL** [comment]
No, this file does not exist anywhere in the profile i used to test. You can probably see in the logs that the game is looking for the file, but is unable to load it. The game CTDs regardles if this file is present or not.
This file is optional for hdt smp to function and is called a registry xml. SMP uses this file to apply physics to defined mesh names in NIFs which do not have a dedicated controller xml file.

**Comment by xKikis** [comment]
Does anyone found a soulition to this problem ? im using MO2 2.4.4.
